### PR TITLE
issue template for new data structure

### LINKED ISSUE #102: Improve issue template
The issue template of this repository is still working with the old data structure. If we want to encourage the community to contribute the mail servers they use to this repository, we should improve on that.

### PR DESCRIPTION
**Pull request by missytake**
closes #102

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -17,25 +17,23 @@ But if you can please provide more information below.
...
-    smtp-server: *
-    smtp-port: *
-    smtp-security: *
-    imap-server: *
-    imap-port: *
-    imap-security: *
+```
+    smtp-server: smtp.example.com
+    smtp-port: 25/465/587...
+    smtp-security: SSL/STARTTLS/NONE
```
Comment (r10s) [inline .github/ISSUE_TEMPLATE.md]: - maybe call it `smtp-socket`, this is also the wording used in the .yml

- `NONE` is currently not an accepted value, only `SSL` or `STARTTLS` are

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -17,25 +17,23 @@ But if you can please provide more information below.
...
 
-### provider is funded by 
-- [ ] advertisements 
-- [ ] recurring payments 
-- [ ] donations 
-- [ ] private/pro-bono for friends 
+    imap-server: imap.example.com
+    imap-port: 993/143...
```
Comment (r10s) [inline .github/ISSUE_TEMPLATE.md]: same here, maybe `imap-socket` is better

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (missytake) [reply .github/ISSUE_TEMPLATE.md]: We can include socket, but this is the contributor-facing issue template, so I would still leave security in. It's not defining it very well, I know, but when I first read socket I didn't know what's meant. "security" in combination with "SSL" and "STARTTLS" should communicate what we mean.

Comment (r10s) [reply .github/ISSUE_TEMPLATE.md]: sounds reasonable.

**Pull request merged**